src/robusta/integrations/kubernetes/custom_models.py

```diff
@@ -74,6 +74,15 @@ def does_node_have_taint(node: Node, taint_key: str) -> bool:
     return any(t.key == taint_key for t in node.spec.taints)
 
 
+class RobustaEvent:
```

Comment:
This should inherit from the Hikaru `Event` class for consistency with other classes like `RobustaPod`.

Can we also name the function `get_events_for_object` or something more indicative?

Reply:
Removed it all together